README.md

```diff
@@ -27,3 +27,4 @@ Avinash V K - avinash-vk
 Varun Seshu - Varun487  
 Pradish Kapur - pk-95  
 Noel Saju - Noel-jediknight
+Anirudh Rowjee - `anirudhRowjee`
```

Comment:
Tut tut
1. Not on a new line (markdown ftw)
2. backticks? Well, if you _really_ wanna stand out, might approve that

Reply:
Right, nvm not gonna wait for you to resolve this